mavericks-db/math-magicians
# Milestone 06: Testing

## Testing

This branch includes:
- [x] Set up React Testing Library.
- [x] Write unit tests for the files `operate.js` and `calculate.js` using Jest.
- [x] Create unit tests for all React components:
   - Use Jest snapshots to test the components.
   - Use React Testing Library to simulate user interaction.
- [x] Run your full tests suite using Jest. All tests should pass.
